Core/Core/Api/Operations/Operations.Send.cs

```diff
@@ -18,17 +18,183 @@ namespace Speckle.Core.Api;
 
 public static partial class Operations
 {
-  #region Pushing objects
+  /// <summary>
+  /// Sends a Speckle Object to the provided <paramref name="transport"/> and the default local <see cref="SQLiteTransport"/>
+  /// </summary>
+  /// <param name="value">The object you want to send</param>
+  /// <param name="transport">Where you want to send them</param>
+  /// <param name="onProgressAction">Action that gets triggered on every progress tick (keeps track of all transports)</param>
+  /// <param name="cancellationToken"></param>
+  /// <returns>The id (hash) of the object sent</returns>
+  /// <exception cref="ArgumentNullException"><paramref name="value"/> or <paramref name="transport"/> was null</exception>
+  /// <exception cref="SpeckleException">Serialization or Send operation was unsuccessful</exception>
+  /// <exception cref="TransportException"><paramref name="transport"/> failed to send</exception>
+  /// <exception cref="OperationCanceledException"><paramref name="cancellationToken"/> requested cancellation</exception>
+  /// <example>
+  /// <code>
+  /// using ServerTransport destination = new(account, streamId);
+  /// string id = await Operations.Send(mySpeckleData, destination);
+  /// </code>
+  /// </example>
+  public static async Task<string> Send(
+    Base value,
+    ITransport transport,
+    Action<ConcurrentDictionary<string, int>>? onProgressAction = null,
+    CancellationToken cancellationToken = default
+  )
+  {
+    if (transport is null)
```

Comment:
Is this necessary anymore? transport isn't nullable, calling this and ignoring the contract would be a bi daft

Reply:
I've changed this with my latest changes, we are no longer doing this check for transports.
However, I've still kept it in for `value`, since I'm not confident that it would throw lower down (we CAN serialize a null) but in the context of a send, we don't want users doing this.

If we were in .NET Standard2.1, we would want to mark the parameter as nullable, but use the `[NotNull]` to say that this function will always throw when a null is passed in. see https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/attributes/nullable-analysis 
We can use Polysharp to bring this attribute back to us, but we haven't got that setup (tho, I believe we're no longer blocked on the CI side)

So for now, I think this is fine.

Reply:
I've changed it again.
The single transport version is truly nullable, with the `useDefaultCache` bool exposed. Now the null check is a feature, not a bug. And with it the DX should be improved.